src/lib/targeting.test.ts

```diff
@@ -54,6 +54,12 @@ describe('shouldNotRenderEpic', () => {
         expect(got).toBe(true);
     });
 
+    it('return false if page is sensitive', () => {
```

Comment:
It should read `return true if page is sensitive` right?

Reply:
ooh yes!